include/qthread/qthread.hpp

```diff
@@ -14,7 +14,11 @@
 template <bool> class OnlyTrue;
 template <> class OnlyTrue<true>
 {};
+#if __cplusplus >= 201103
```

Comment:
Ah, I didn't know that about the preprocessor. I assumed it was either like C, where a non-defined value is a syntax error (I actually think I've run into that problem before, and been forced to wrap things inside an #ifdef)